### DISCUSSION TITLE
Application Issues

### DISCUSSION THREAD

**Discussion by ukmartin** [valinet/ExplorerPatcher#1761] [General]
Is it just me or does Explorer Patcher create issues with some applications which need to browse to a file to open them?

To name two - Handbrake (the media file converter / compressor) and Photomatix (HDR Photo Processor) both repeatedly crashed whenever I went into them to browse for files to process. It was a case of FILE > Open > POOF! Application gone. I can only assume that something in the process of calling Explorer from within those applications was screwing up. If it wasn't for being able to drag'n'drop the files, those applications would have been rendered useless to me whilst Explorer Patcher was installed. 

Now that Explorer Patcher has been uninstalled, both applications work just fine again.

Is this an issue that others are having and is it worth waiting for a future version of Explorer Patcher to come out to try again?

**Comment by Jamesstevens-3rd** [comment]
I use handbrake almost daily and cannot reproduce nor have I countered this behaviour, do you have the latest version installed? what happens if you drag-n-drop a file onto the interface?
1.6.1
https://handbrake.fr/downloads.php

**Comment by ukmartin** [comment]
I'm using 1.6.4 (2023012300) As I said, dragging and dropping wasn't a problem. Go anywhere near the "Open Source" button though, and you're on a countdown to destruction. Additionally, clicking on the Browse button to find somewhere to save the output file also has the same effect of crashing the application. The common feature here is that both invoke Explorer in one way or another. Manually typing in the path to the save location was fine, it had no problems with saving it where I wanted it.

Yeah, it's a weird one.  Another casualty was the Snipping tool, which refused to save it's snips as it too couldn't surf to a saving location. Snipping Tool's "crash" was weird. I just got error messages saying that the file could not be saved, and the application just hung there. It had to be closed down from Task Manager. In the end, I learned to copy the clipboard content to Paint (or similar) and save the file from there, as Paint was unaffected.

As I said though, all is well again after uninstalling Explorer Patcher.

________________________________

**Comment by pyrates999** [comment]
Try the following:

1. uninstall explorer patcher
2. delete the registry key: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher"
3. reboot
4. install the latest explorer patcher
5. verify you get a notification that symbols have been downloaded
6. restart windows explorer using either the task manager or in the properties window of explorer patcher
7. reboot

Then apply your settings again in explorer patcher to see if it fixes it.

**Comment by ukmartin** [comment]
Thanks, will give it a try as soon as my current recode job is complete.

:-)

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by ukmartin** [comment]
Thanks, the latest version seems to be working fine so far.

One little thing though, more of a request than an issue...having retured to with Win11 world for a few weeks, I've found how useful it is to open WIndows Explorer in a new tab, like I do with web browsers. Is this a feature that can be made an option for future versions of Explorer Patcher?

Thanks

Martin
________________________________

**Comment by pyrates999** [comment]
That would be something Microsoft adds.

**Comment by ukmartin** [comment]
It's IN the latest version of Windows 11 context menu, I think it was added in H22H1 but it's stripped out in Explorer Patcher.
________________________________

**Comment by pyrates999** [comment]
Oh, then I would open up that under a new discussion.  Or maybe even issues if you think it's a bug.
